pkg/project/resource_project_repo_test.go

```diff
@@ -2,11 +2,11 @@ package project
 
 import (
 	"fmt"
-	"strings"
-	"testing"
-
 	"github.com/go-resty/resty/v2"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"strconv"
+	"strings"
+	"testing"
```

Comment:
Whichever tool you use to format the code file, it is not doing what we want. The convention now is to groups in following order (per `goimports`):
- Golang builtin packages
- 3rd party packages

You can change how Goland sort imports: https://stackoverflow.com/a/70668482

Reply:
done and changed my go lang settings as well